docs/apis/subsystems/communication/index.md

```diff
@@ -32,6 +32,14 @@ Please follow the steps to enable the feature.
 
 :::
 
+## Hooks
+
+The Communication API takes advantage of the [Hooks API](../../plugintypes/communication/index.md) with actions performed in the following way:
+
+1. Actions in core have their own hook dispatched (e.g. enrol, change role, add to group).
+2. Hooks are then registered to a particular callback (`lib/db/hooks.php`).
```

Comment:
As this is a generic location where more hooks will be stored (not just the ones affecting communication), shouldn't it be mentioned explaining how to identify the communication ones? ;-)

Reply:
Hi @sarjona 
I didn't create any communication-specific hooks, for example, hooks related to enrolment were, before enrolment, after enrolment, they were all generic. 
I have added a forth point to mention that.